### ISSUE TITLE
Inconsistencies in symbol pronunciation when reading by word and encountering a "word" which only contains multiple symbols.

### ISSUE THREAD

**Issue by Qchristensen** [nvaccess/nvda#11779]
### Steps to reproduce:

1. Open Word, or NotePad++ and type the line of text:
He said (quietly), "Hello, Jim".
Note the parenthesis either side of quietly, followed by a comma, and the quotes either side of "Hello Jim" followed by a full stop.
2. Press NVDA+P until the symbol level is set to "none".
3. Press HOME to move to the start of the line, then use NVDA+right arrow to move through the sentence by word.

### Actual behavior:

NVDA sends different information to the synthesizer, depending on the context of a symbol.  This can result in, for instance "left paren" being read, but not its right counterpart.  Or in the case of the quote, it is read once (where it is followed by a word), but not the second time (where it is followed by a full stop).
When moving by word, NVDA sends the following to the synthesizer (copied from the speech viewer:

He
said 
left paren
quietly
), 
quote
Hello
comma
Jim
".
blank

Because NVDA encounters the left paren and first quote as a single symbol on their own, the text "left paren" and the first "quote" are sent as words, and they are read.  Because ), and ". are sent as symbols (which are not set to be read at symbol level none) they are not read.

### Expected behavior:

NVDA should consistently read symbols (and either read them or not - but not read some and not others at the same symbol level).  Left and right paren are both symbol level "most" by default, as is quote.

#### NVDA version:

NVDA 2020.3

#### Windows version:

Windows 10 (64-bit) Version: 2004 Build 19041 

#### Name and version of other software in use when reproducing the issue:

Office 365 (64-bit) Version: 16.0.13231.20372 
NotePad++ v7.9

#### Other information about your system:

I was going to use Notepad, but it moves by word differently.  Where Notepad++ and Word both move to the start of the next word or group of symbols (a group may be one), Notepad simply moves to the first character after the next space, so doesn't read any characters in this case.  So, in Notepad, symbols are read only at the level they are set to read at - just as in Word or Notepad++ using Say all.  When moving by one character at a time, every character is read regardless of symbol level, as expeced.  The problem is when moving by word, and the whole "word" to be read consists of more than one symbol, they are treated by their symbol level.

I think the best solution is to ignore symbol level when moving by word.  Word is granular enough that it is worth knowing all the symbols you pass.  If reading by line or greater, then read according to symbol level.  The only case caught in the middle then would be a case where a line contains only symbols, which is a much rarer case than encountering a "word" of only symbols when moving by word.

#### Does the issue still occur after restarting your computer?
Yes
#### Have you tried any other versions of NVDA? If so, please report their behaviors.
I don't think this behaviour has changed recently.  I tried NVDA 2019.2.1 portable and it behaves the same.

#### If addons are disabled, is your problem still occuring?
Yes
#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
No

**Comment by michaelDCurran** [comment]
I agree this is confusing. My feeling is that we should hardcode symbol level to all for word and character.
This would be in contrast to pr #11167

**Comment by JulienCochuyt** [comment]
@michaelDCurran wrote:
> My feeling is that we should hardcode symbol level to all for word and character.

We agree here at Accessolutions that this sounds like the most suitable behavior.

**Comment by JulienCochuyt** [comment]
Please find an implementation proposal as PR #11856.
Next step in this scope would be, IMHO, to ensure NVDA never remains silent at all when the caret moves, eg. when moving by line and landing on a line with only unspoken symbols. ("blank" would be a plus, the actual symbols a best, silence is always unwanted)

**Comment by amirsol81** [comment]
@Qchristensen @JulienCochuyt @michaelDCurran Why not extend this great change to selecting text via Control+SHIFT+Left, Control+SHIFT+Right, SHIFT+Home and SHIFT+End? Text selection also suffers from the very same issues described here.

**Comment by feerrenrut** [comment]
#12695 reverts #11856 thus re-opens this issue.

**Comment by Adriani90** [comment]
cc: @jcsteh here is an interesting discussion with some referenced PRs regarding hardcoding of reporting of symbols and punctuation when using character and word navigation.
